idl/partition_checksum.idl.hh

```diff
@@ -142,3 +142,16 @@ struct node_ops_cmd_response {
     // Optional field, set by query_pending_ops cmd
     std::list<utils::UUID> pending_ops;
 };
+
+struct repair_update_system_table_request {
+    utils::UUID repair_uuid;
+    utils::UUID table_uuid;
+    sstring keyspace_name;
+    sstring table_name;
+    dht::token_range range;
+    gc_clock::time_point repair_time;
+};
+
+struct repair_update_system_table_response {
+    bool ok;
```

Comment:
Do we need this ok? If it fails it can throw.

Reply:
It is not needed. Yes, we can throw, but rpc can not send back the exception type info. Still remember, we need to send the error code for rpc streaming in the high level?  I want to keep the repair_update_system_table_response struct  so we can extending the respond more easily in the future.  

Do you want to me to drop the ok field and send an empty repair_update_system_table_response? Or just keep it?

Reply:
Empty response is better IMO. With "ok", we need to check both for exceptions and "ok", and what do we do if not ok? We just throw an exception with no good message.

Reply:
All right.